fix: use tables for tabular data and add table styles

HendrikThePendric opened the pull request:
I've also left aligned the labels because this was hurting my eyes. IMO this is looking much better now:
PRE:
![Screenshot 2020-09-17 at 16 37 53](https://user-images.githubusercontent.com/353236/93486201-33a72980-f904-11ea-934a-3c6023533d54.png)


POST:
![Screenshot 2020-09-17 at 16 37 33](https://user-images.githubusercontent.com/353236/93486222-399d0a80-f904-11ea-8865-c413cc5f28ad.png)

ismay reviewed (approved):
Nice, looks good 👍. Two things that stand out to me:

* I find it slightly confusing that the `InfoItem` is returning a tablerow. It needs to be embedded in a table element, but the naming doesn't clarify that immediately to me.
* It looks like the line height is a little smaller in the table, compared to the original styles. Maybe we could up that a little for readability.

HendrikThePendric commented:
> * I find it slightly confusing that the `InfoItem` is returning a tablerow. It needs to be embedded in a table element, but the naming doesn't clarify that immediately to me.

Good point, renamed it to `InfoRow`

> * It looks like the line height is a little smaller in the table, compared to the original styles. Maybe we could up that a little for readability.

Good find, that was a regression

I've addressed both issues in 3f329ae
